cloud-pubsub/main.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Pub/Sub pull example on Google Container Engine.
+
+This program pulls messages from a Cloud Pub/Sub topic and
+prints to standard output.
+"""
+
+import datetime
+from google.cloud import pubsub
+from google.cloud.pubsub.subscription import AutoAck
```

Comment:
our style guide strongly prefers import modules instead of importing the members inside of them.